src/FormatHelper/StripCountryCode.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brick\Postcode\FormatHelper;
+
+/**
+ * Strip country code prefix from postalcode if it is valid.
+ *
+ * In some countries like Belgium or Luxembourg its common to provide a country code prefix in a postalcode,
+ * Although the format is incorrect, many government agencies, businesses and people still use this.
+ * Rather than failing this format into an exception, this helper class strips the prefix off before validating it,
+ * Thus providing you with the correct format without the country code prefix in it.
+ *
+ * @see https://en.wikipedia.org/wiki/Postal_codes_in_Belgium
+ */
+trait StripCountryCode
+{
+    /**
+     * @param string $postcode
+     * @param int $validCountryCodeLength
+     * @return string
+     */
+    public function stripCountryCode(string $postcode, int $validCountryCodeLength = 1): string
+    {
+        $code = substr((new \ReflectionClass($this))->getShortName(), 0, $validCountryCodeLength);
```

Comment:
@BenMorel I don't see why you would not want to rely on Reflection or a defined constant in the class maybe but to prevent a long discussion obstructing this from releasing (which is what i need) i implemented your solution. Which seems to work nicely.